benchmark/opperf/nd_operations/nn_conv_operators.py

```diff
@@ -60,131 +81,286 @@ def run_pooling_operators_benchmarks(ctx=mx.cpu(), dtype='float32', profiler='na
+                                                                   profiler=profiler,
+                                                                   inputs=[{"data": conv_data,
+                                                                            "weight": (1, 1, 3),
+                                                                            "bias": (1,),
+                                                                            "kernel": (3,),
+                                                                            "stride": (1,),
+                                                                            "dilate": (1,),
+                                                                            "pad": (0,),
+                                                                            "num_filter": 1,
+                                                                            "no_bias": False,
+                                                                            "layout": 'NCW'}
+                                                                          ],
+                                                                   warmup=warmup,
+                                                                   runs=runs)
+        # Conv2DTranspose Benchmarks
+        conv2d_transpose_benchmark_res = []
+        for conv_data in [(2**29, 1, 3, 3), (2**28, 1, 4, 4)]:
+            conv2d_transpose_benchmark_res += run_performance_test([getattr(MX_OP_MODULE, "Deconvolution")],
+                                                                   run_backward=True,
+                                                                   dtype=dtype,
+                                                                   ctx=ctx,
+                                                                   profiler=profiler,
+                                                                   inputs=[{"data": conv_data,
+                                                                            "weight": (1, 1, 3, 3),
+                                                                            "bias": (1,),
+                                                                            "kernel": (3, 3),
+                                                                            "stride": (1, 1),
+                                                                            "pad": (0, 0),
+                                                                            "num_filter": 1,
+                                                                            "no_bias": False,
+                                                                            "layout": 'NCHW'}
+                                                                          ],
+                                                                   warmup=warmup,
+                                                                   runs=runs)
+    else:
```

Comment:
It seems the only difference between the if and else branch is the `inputs` argument. Can we only generate different inputs in the if/else branch and pass them to the same operator function?